andela/rivendell-ah-client
# #160056383: user shares article via social

#### What does this PR do?
- adds feature that allows users to share article on social media
- creates a SocialShare component
- updates tests and ensures that all the tests are passing

#### Description of Task to be completed?
- user should be able to share articles on social platform

#### How should this be manually tested?
- `git pull` this branch
- update environmental variables using the .env-sample as a reference
- npm start to start the server
-  view an article via `/articles/{slug}`
- underneath the page you would see the Share article section

#### Any background context you want to provide?
- None
#### What are the relevant pivotal tracker stories?
- 160056383
#### Screenshots (if appropriate)
<img width="1440" alt="screen shot 2018-09-26 at 1 26 18 pm" src="https://user-images.githubusercontent.com/38565349/46079715-d43c3600-c18f-11e8-88d6-51090bdc5025.png">
<img width="1440" alt="screen shot 2018-09-26 at 1 26 17 pm" src="https://user-images.githubusercontent.com/38565349/46079716-d43c3600-c18f-11e8-9f4e-eb5506495732.png">
<img width="1440" alt="screen shot 2018-09-26 at 1 26 13 pm" src="https://user-images.githubusercontent.com/38565349/46079717-d43c3600-c18f-11e8-8039-182aa53b67a5.png">
<img width="1440" alt="screen shot 2018-09-26 at 1 26 04 pm 1" src="https://user-images.githubusercontent.com/38565349/46079718-d4d4cc80-c18f-11e8-890f-51cceaaf9a6e.png">
<img width="1440" alt="screen shot 2018-09-26 at 1 26 04 pm" src="https://user-images.githubusercontent.com/38565349/46079721-d4d4cc80-c18f-11e8-82ba-92fcaa949e91.png">
<img width="1440" alt="screen shot 2018-09-26 at 1 25 54 pm" src="https://user-images.githubusercontent.com/38565349/46079724-d4d4cc80-c18f-11e8-8dda-bcfba45c9a52.png">
<img width="1440" alt="screen shot 2018-09-26 at 1 25 16 pm" src="https://user-images.githubusercontent.com/38565349/46079725-d56d6300-c18f-11e8-9f24-f37f93fa3804.png">
<img width="1440" alt="screen shot 2018-09-26 at 1 25 03 pm" src="https://user-images.githubusercontent.com/38565349/46079727-d56d6300-c18f-11e8-82b6-167cccb2647d.png">
<img width="1440" alt="screen shot 2018-09-26 at 1 24 53 pm" src="https://user-images.githubusercontent.com/38565349/46079730-d605f980-c18f-11e8-8cde-1bd37e32f9e2.png">
<img width="1440" alt="screen shot 2018-09-26 at 1 24 19 pm" src="https://user-images.githubusercontent.com/38565349/46079750-ea49f680-c18f-11e8-8652-1bc295f534e6.png">
<img width="1440" alt="screen shot 2018-09-26 at 1 24 18 pm" src="https://user-images.githubusercontent.com/38565349/46079752-eae28d00-c18f-11e8-81ff-d9014f22d468.png">
<img width="1440" alt="screen shot 2018-09-26 at 5 08 49 am" src="https://user-images.githubusercontent.com/38565349/46079753-eb7b2380-c18f-11e8-8eb2-a5a2f615b3be.png">
<img width="1440" alt="screen shot 2018-09-26 at 1 28 33 pm" src="https://user-images.githubusercontent.com/38565349/46079822-1796a480-c190-11e8-8ec3-a7309b343291.png">

#### Questions:
- None
